src/HTSPDemuxer.cpp

```diff
@@ -159,21 +162,28 @@ bool CHTSPDemuxer::Seek
   if (!m_subscription.IsActive())
     return false;
 
-  if (!m_subscription.SendSeek(time))
+  m_seeking = true;
+  if (!m_subscription.SendSeek(time)) {
```

Comment:
Can you do me a favour and put the curly bracket one line down? I hate this coding style.... Just asking.